### ISSUE TITLE
Cannot see the MIME type of a video when editing its info from UI

### ISSUE THREAD

**Issue by fpoto** [gerbera/gerbera#872]
Gerbera 1.5.0 from deb-multimedia.org

From the UI I choose edit, a windows pops up but it's mostly empty. The only info is the first field, Typr, which is set to Container, all other fields are empty. Specifically, I was looking for the MIME type, in order to understand why my TV cannot decode some videos and possibly set up transcoding.

These are the logs (--debug) just after pressing the edit button:
```
May 15 16:46:55 pot gerbera[2446436]: 2020-05-15 16:46:55  debug: [server.cc:490] createRequestHandler(): Filename: /content/interface?req_type=edit_load&sid=9e3915ca-ede6-464e-8756-91dc9d6d256d&object_id=21&updates=check
May 15 16:46:55 pot gerbera[2446436]: 2020-05-15 16:46:55  debug: [upnp_headers.cc:63] addHeader(): Adding header: 'Cache-Control: no-cache, must-revalidate'
May 15 16:46:55 pot gerbera[2446436]: 2020-05-15 16:46:55  debug: [server.cc:490] createRequestHandler(): Filename: /content/interface?req_type=edit_load&sid=9e3915ca-ede6-464e-8756-91dc9d6d256d&object_id=21&updates=check
May 15 16:46:55 pot gerbera[2446436]: 2020-05-15 16:46:55  debug: [web_request_handler.cc:222] open(): output-----------------------{"success":true,"item":{"object_id":21,"title":{"value":"La fame e la sete - Antonio Albanese 1999.avi","editable":1},"class":{"value":"object.item.videoItem","editable":true},"obj_type":"item","description":{"value":"","editable":true},"location":{"value":"/casapot/video/La fame e la sete - Antonio Albanese 1999.avi","editable":false},"mime-type":{"value":"video/x-msvideo","editable":true}},"update_ids":{"pending":false}}
May 15 16:46:56 pot gerbera[2446436]: 2020-05-15 16:46:56  debug: [server.cc:490] createRequestHandler(): Filename: /content/interface?req_type=void&sid=9e3915ca-ede6-464e-8756-91dc9d6d256d&updates=get
May 15 16:46:56 pot gerbera[2446436]: 2020-05-15 16:46:56  debug: [upnp_headers.cc:63] addHeader(): Adding header: 'Cache-Control: no-cache, must-revalidate'
May 15 16:46:56 pot gerbera[2446436]: 2020-05-15 16:46:56  debug: [server.cc:490] createRequestHandler(): Filename: /content/interface?req_type=void&sid=9e3915ca-ede6-464e-8756-91dc9d6d256d&updates=get
May 15 16:46:56 pot gerbera[2446436]: 2020-05-15 16:46:56  debug: [web_request_handler.cc:222] open(): output-----------------------{"success":true,"update_ids":""}
```

**Comment by whyman** [comment]
You can see that the server is sending that info the UI: `"mime-type":{"value":"video/x-msvideo","editable":true}}`, do you get any errors in the web browser dev console?

**Comment by fpoto** [comment]
Ah, I had not noticed it! So it looks like a client problem!

First, it is a Debian packaging bug of the related package `libjs-popper.js`.
I have filed Debian bug https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=960725
In the meantime, a workaround is to do:
# rm /usr/share/gerbera/web/vendor/popper/popper.js
# ln -s /usr/share/nodejs/popper.js/dist/popper.js /usr/share/gerbera/web/vendor/popper

After that, the Firefox debug log showed an error on popper.js, so I downloaded it from https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js

Now the Firefox debug log  does not show errors any more, but the popup windows is still empty. Same with the Chromium browser and Firefox on Android. However, I don't know where to look any more :(

**Comment by KarlStraussberger** [comment]
The file popper.js is included in gerbera. Replacing it with another version may break the ui. Please restore the original and post your browser console.

**Comment by fpoto** [comment]
Found the culprit. The gerbera 1.5.0 package on deb-multimedia decided to use a shared version of several javascript packages, including bootstrap.js and popper.js. Removing the whole /usr/share/web directory and substituting it with the one on the trunk solved most UI problems.
In short, this is a packaging bug.

**Comment by whyman** [comment]
I guess you will need to raise this problem with @dmo

**Comment by whyman** [comment]
I guess you will need to raise this problem with @marillat

**Comment by fpoto** [comment]
Marillat writes: 
Fixed in gerbera 1.5.0-dmo3

This also fixes #868
